core/vm/contracts.go

```diff
@@ -217,6 +220,48 @@ var (
 	big199680 = big.NewInt(199680)
 )
 
+// multComplexity implements bigModexp multComplexity formula, as defined in EIP-198
+//
+// def mult_complexity(x):
+//    if x <= 64: return x ** 2
+//    elif x <= 1024: return x ** 2 // 4 + 96 * x - 3072
+//    else: return x ** 2 // 16 + 480 * x - 199680
+//
+// where is x is max(length_of_MODULUS, length_of_BASE)
+func multComplexity(x *big.Int) *big.Int {
```

Comment:
Please give a more meaningful name to this method. `vm.multComplexity` seems dangling. Maybe add ModExp somewhere or maybe we could make it a method on `bigModExp`?